form_designer/models.py

```diff
@@ -152,7 +152,10 @@ def send_mail(self, form, files=None):
 
     @property
     def is_template_html(self):
```

Comment:
Okay, getting rid of the `HTML_TEMPLATE` option was a good idea too!

However, I think this function is more clearly expressed as

```python
    @property
    def is_template_html(self):
        template = self.message_template
        if template:  # We have a custom inline template string?
        	# Assume the template string is HTML-ish if it has at least one opening
        	# and closing HTML tag:
        	return (re.search(u"<[^>]+>", template) and re.search(u"</[^>]+>", template))

        # If there is no custom inline template, see if the `EMAIL_TEMPLATE`
        # setting points to a `.html` file:
        return settings.EMAIL_TEMPLATE.lower().endswith('.html')
```